source/administration/migrating.rst

```diff
@@ -75,7 +75,7 @@ The following limitations are present when importing from Slack:
 
 * The Markdown subset used by Slack's Posts 2.0 feature announced on September 28, 2015 is not yet supported.
 
-* Direct Message and Private Groups cannot be imported. Slack does not include these messages when generating the export archive.
+* Direct Message and Private Channels cannot be imported. Slack does not include these messages when generating the export archive.
```

Comment:
"Direct Message" is singular while "Private Channels" is plural...  Could we rather say "Direct Message and Private Channel data..." ?
or
"Direct Messages and Private Channels cannot..."?